.gitignore

```diff
@@ -21,6 +21,7 @@ doc
 Gemfile.lock
 .bundle
 vendor/ruby
+CHANGELOG.md
```

Comment:
generally i think changelog files are tracked, is there a reason this was added?